### PR TITLE
Va 68 added Cypress for e2e testing

### PR DESCRIPTION
**Pull request by shewood**
[VA-68](https://jira-dev.bdm-dev.dts-stn.com/browse/VA-68)

renamed the jest tests from jesttestname.spec.js to jesttestname.test.js

Added Cypress for e2e testing. 
You will need to run npm i. 

Added a few basic tests to start with for the home page and the splash page.

### CODE REVIEW

File: cypress/integration/spalsh.spec.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef */
```
Comment (nibivi77) [inline cypress/integration/spalsh.spec.js:1]: Typo in this file name, it should be splash.spec.js and not spalsh.spec.js :)

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (nibivi77) [inline cypress/support/commands.js:1]: Will we be using this file? i am not very experienced with cypress so I see a commented out file, I question it. :p hehehe

File: cypress/plugins/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```
Comment (nibivi77) [inline cypress/plugins/index.js:1]: I am not 100% sure what this file will be used for. Should we keep it or remove it?

File: cypress/support/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (nibivi77) [inline cypress/support/index.js:1]: Same thing here, what is an example of when we would use this file?

**Review by nibivi77 — COMMENTED** [review]
Mostly comments and questions as I am not very familiar with Cypress.

File: cypress/integration/spalsh.spec.js (hunk 1 above)
Comment (shewood) [reply cypress/integration/spalsh.spec.js:1]: haha, that's what you get for working late I guess...

File: cypress/support/commands.js (hunk 2 above)
Comment (shewood) [reply cypress/support/commands.js:1]: yes, we will use this file, just not yet.

File: cypress/plugins/index.js (hunk 3 above)
Comment (shewood) [reply cypress/plugins/index.js:1]: keep it.

File: cypress/support/index.js (hunk 4 above)
Comment (shewood) [reply cypress/support/index.js:1]: yup, we will be using this soon.

File: cypress/integration/splash.spec.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef */
+/// <reference types="Cypress" />
+
+describe("home page", () => {
```
Comment (nibivi77) [inline cypress/integration/splash.spec.js:4]: Just need to change this to splash page.

**Review by nibivi77 — APPROVED** [review]
Looks good! Just a small typo in the splash page test file.

File: cypress/integration/splash.spec.js (hunk 5 above)
Comment (shewood) [reply cypress/integration/splash.spec.js:4]: oops, copy paste error

**Pull request merged**